src/dev/flang/parser/Parser.java

```diff
@@ -2175,19 +2176,23 @@ Expr term()
             | '}any chars{' block stringTermB
             ;
   */
-  Expr stringTerm(Expr leftString)
+  Expr stringTerm(Expr leftString, Optional<Integer> multiLineIndentation)
   {
     return relaxLineAndSpaceLimit(() -> {
         Expr result = leftString;
         var t = current();
         if (isString(t))
           {
-            var str = new StrConst(posObject(), string());
+            var ps = string(multiLineIndentation);
+            var str = new StrConst(posObject(), ps._v0);
             result = concatString(posObject(), leftString, str);
             next();
             if (isPartialString(t))
               {
-                result = stringTerm(concatString(posObject(), result, block()));
+                var old = minIndent(-1);
```

Comment:
This means that the code in braces inside of a multiline string is allowed to not be indented at all.
Maybe this is not what we want?

Reply:
I think this is fine, let's play around with it. The alternative would be to use the indentation level before the string or of the string itself.